src/corva/models/stream/stream.py

```diff
@@ -16,7 +16,7 @@ class StreamTimeRecord(CorvaBaseEvent):
     """
 
     timestamp: int
-    data: dict = {}
+    data: Optional[dict] = {}
```

Comment:
User-facing events should not be changed, because this change is not backward-compatible

Reply:
This is addressed.